conda_forge_tick/git_utils.py

```diff
@@ -193,7 +193,8 @@ def _run_git_command(
         """
         git_command = ["git"] + cmd
 
-        logger.debug(f"Running git command: {git_command}")
+        if not suppress_all_output:
+            logger.debug(f"Running git command: {git_command}")
```

Comment:
I prefer sticking to the `%s` format in logging calls, but I see that there's a precedent already so I won't repeat this again. Up to your choice.